test/com/minseungkwon/datastructure/Graph/BinaryTreeTest.java

```diff
@@ -116,4 +116,80 @@ public void test_BSTtoList() {
 
         assertEquals("4/17/0258/369/", bt.BSTtoList());
     }
+
+    @Test
+    public void test_isBalancedUsingTwoRecursiveTrueCase() {
+        Integer[] arr = new Integer[10];
```

Comment:
I think you can pull L122-L127 to a method so we can reduce redundancy.

Reply:
Made new method